tests/0010_Integration/CloneTest.php

```diff
@@ -46,11 +46,14 @@ public function testCloneSection() {
 
       //get cloned section
       $originalId = $section->getID();
-      $new_section->getFromDBByQuery("WHERE `name` = 'test clone section' AND `id` <> '$originalId'");
-
-      // check uuid
-      $this->assertNotEquals($section->getField('uuid'),
-                             $new_section->getField('uuid'));
+      $new_section->getFromDBByCrit([
+           'AND' => [
+               'name' => 'test clone section',
+               'NOT' => ['uuid' => $section->getField('uuid')], // operator <> available in GLPI 9.3+ only
+               'plugin_formcreator_forms_id' => $section->getField('plugin_formcreator_forms_id')
```

Comment:
if an other unit test creates a section with the same name, the previous query may fail. Checking only among the sections of the same form as the cloned section